src/stashChecker.ts

```diff
@@ -363,6 +363,16 @@ export async function runStashChecker() {
             check(Target.Scene, "[class*='pcVideoListItem'] span.title a[href*='/view_video.php?viewkey=']", { observe: true })
             break;
         }
+        case "www.clips4sale.com": {
+            check(Target.Studio, "div[class*='items-center'] h1", { urlSelector: currentSite });
+            check(Target.Studio, "div[class*='items-center'] div[class*='flex']:not([class*='hidden']) a[href*='/studio/']", {
```

Comment:
This seems to match the title as well, resulting in additional queries that will be overwritten by studio queries. Maybe use `[data-testid*='studioAnchor'] a[href*='/studio/']` instead.